taglibs/async/await-tag-transformer.js

```diff
@@ -3,27 +3,36 @@
 var isObjectEmpty = require('raptor-util/isObjectEmpty');
 
 module.exports = function transform(el, context) {
-    var varName = el.getAttributeValue('var');
-    if (varName) {
-        if (varName.type !== 'Literal' || typeof varName.value !== 'string') {
-            context.addError(el, 'The "var" attribute value should be a string');
-            return;
-        }
+    if(!el.argument) {
+        context.addError(el, 'Invalid <await> tag. Argument is missing. Example: <await(user from data.userProvider)>');
+        return;
+    }
 
-        varName = varName.value;
+    var parts = el.argument.split(' from ');
```

Comment:
Using split is not a good idea since there is very small chance ` from ` might appear in the expression. I would use a regular expression that is rooted on the side of the variable name to separate out the variable name from the potentially complex expression:

```javascript
var matches = / from ([$A-Z_][0-9A-Z_$])$/i.exec(argument);
```